samples/Sandbox/Dependency.cs

```diff
@@ -10,14 +9,14 @@ public class Dependency : IDependency, IDisposable
 
         public Dependency(ILogger<Dependency> logger)
         {
-            this._logger = logger;
+            _logger = logger;
```

Comment:
I don't mind little optimizations and things, but I _am_ a fan of putting `this.` in front of instance items for clarity. It also makes code reviews kinda large with unrelated changes. Something to consider.

Reply:
Maybe at some point I need to add updated .editorconfig and analyzer ruleset settings to fix this stuff. Otherwise there's going to be the same sort of code churn of add/remove/add/remove as different preferences switch these things. Hmmm.

Reply:
I am also a friend of this to be honest. I usually don't use underscore for the fields then though. I will read them!